src/Microsoft.AspNetCore.Hosting/project.json

```diff
@@ -50,7 +52,7 @@
         }
       }
     },
-    "netstandard1.3": {
+    "netstandard1.5": {
```

Comment:
I understand that netstandard is a good thing but it's also confusing as hell... isn't net451 here just for backwards-compatibility until all tools understand netstandard? why would it be necessary to have both if netstandard covers the full .net framework anyway? (sorry, not the best place to ask this but since you are the guru, I'm asking it here anyway :) )